src/widgets/Header.js

```diff
@@ -0,0 +1,59 @@
+import userService from "../features/UserService";
+import { BASE_URL } from "../shared/const";
+import { router } from "../app";
+
+const Header = () => {
+  const LOGOUT = "로그아웃";
+  const navItemList = [
+    { href: "/", title: "홈" },
+    { href: "/profile", title: "프로필" },
+    userService.isLoggedIn()
+      ? { href: "/login", title: LOGOUT }
+      : { href: "/login", title: "로그인" },
+  ];
+
+  const currentPath =
+    window.location.hash?.replace("#", "") || window.location.pathname;
+
+  const isCurrNav = (path) => path === currentPath;
+  const navTextColor = (path) => (isCurrNav(path) ? "blue-600" : "gray-600");
+  const navFontWeight = (path) => (isCurrNav(path) ? "bold" : "");
+
+  const view = `
+      <header class="bg-blue-600 text-white p-4 sticky top-0">
+        <h1 class="text-2xl font-bold">항해플러스</h1>
+      </header>
+      <nav class="bg-white shadow-md p-2 sticky top-14">
+        <ul class="flex justify-around">
+        ${navItemList
+          .map(({ href, title }) => {
+            return `<li><a href="${href}" id="${title === LOGOUT ? "logout" : ""}" class="text-${navTextColor(href)} font-${navFontWeight(href)}">${title}</a></li>`;
+          })
+          .join("")}
+        </ul>
+      </nav>
+    `;
+
+  const init = () => {
+    const nav = document.querySelector("nav");
+
+    nav.addEventListener("click", (e) => {
+      e.preventDefault();
+
+      if (e.target.innerHTML === LOGOUT) {
```

Comment:
innerHTML 는 "로그아웃" 이라는 문자열 대신 다른 내용(이를테면 단순 문자열이 아니라 dom <b>logout</b> 과 같은,,)으로 내용이 바뀌면 로직이 동작하지 않을 것 같습니다. 아마도 불변할 id 값을 기준으로 동작시키면 변경에 더 안전할 것 같아요

```suggestion
      if (e.target.id === LOGOUT) {
```